plugins/helm/src/main/scala/org/broadinstitute/monster/sbt/MonsterHelmPlugin.scala

```diff
@@ -68,29 +68,78 @@ object MonsterHelmPlugin extends AutoPlugin {
       }
       targetDir
     },
+    releaseHelmChart := {
+      val log = streams.value.log
+      val packageDir = packageHelmChart.value.getAbsolutePath()
+      val org = helmChartOrganization.value
+      val repo = helmChartRepository.value
+      val indexTarget = helmChartLocalIndex.value
+
+      // Assumes chart-releaser is available on the local PATH,
+      // and that CR_TOKEN is in the environment.
+      def cr(cmd: String, args: Map[String, String]): Unit = {
+        val fullCommand =
+          "cr" :: cmd :: args.toList.flatMap { case (k, v) => List(k, v) }
+        val result = Process(fullCommand).!
+        if (result != 0) {
+          sys.error(s"`cr $cmd` failed with exit code $result")
+        }
+      }
+
+      log.info(s"Uploading Helm chart for ${name.value} to $org/$repo...")
+      cr("upload", Map("-o" -> org, "-r" -> repo, "-p" -> packageDir))
+
+      val indexSite = s"https://$org.github.io/$repo"
+      log.info(s"Adding Helm chart for ${name.value} to index for $indexSite...")
+      cr(
+        "index",
+        Map(
+          "-c" -> indexSite,
+          "-o" -> org,
+          "-r" -> repo,
+          "-p" -> packageDir,
+          "-i" -> indexTarget.getAbsolutePath()
+        )
+      )
+
+      val gitBase = (ThisBuild / baseDirectory).value
+
+      def git(cmd: String, args: String*): Unit = {
```

Comment:
any point in having a more generic function that `cr` and `git` inherit from/are based off of? Structurally similar and both have the exit code check, but I understand that this is pretty tightly scoped rn so there might not be too much value in doing that. Just a nitpick 🙂

Reply:
I think there's enough difference that trying to consolidate the two wouldn't really be worth it.